Sources/swift-parsing-benchmark/JSON.swift

```diff
@@ -35,8 +35,10 @@ let jsonSuite = BenchmarkSuite(name: "JSON") { suite in
       string.append(contentsOf: fragment)
     } element: {
       OneOf {
-        Prefix(1...) { $0 != .init(ascii: "\"") && $0 != .init(ascii: "\\") }
-          .map { String(Substring($0)) }
+        Prefix(1...) {
+          $0 != .init(ascii: "\"") && $0 != .init(ascii: "\\") && $0 >= .init(ascii: " ")
```

Comment:
This is "better" JSON parsing logic (control character like newlines are invalid). Before this change, the `Prefix(1...)` would push the failure onto the next line, which is more difficult to read/debug.